modules/light-clients/08-wasm/types/utils.go

```diff
@@ -52,3 +52,20 @@ func (l *limitedReader) Read(p []byte) (n int, err error) {
 	}
 	return l.r.Read(p)
 }
+
+// GzipIt compresses the input ([]byte)
```

Comment:
same upstream

Reply:
same upstream? 

What do you mean? Personally I'd like to see these functions copied from `x/wasm` moved into their own `ioutil` pkg

Reply:
ah yea, upstream indicating wasmd here.